include/aikido/statespace/dart/MetaSkeletonStateSpace.hpp

```diff
@@ -171,6 +171,12 @@ class MetaSkeletonStateSpace : public CartesianProduct
   void convertStateToPositions(
       const State* _state, Eigen::VectorXd& _positions) const;
 
+  /// Returns a \c State in this state space that corresponds to
+  /// specified DART \c MetaSkeleton positions.
+  ///
+  /// \param[in] positions DART \c MetaSkeleton positions
+  ScopedState getStateFromPositions(Eigen::VectorXd& positions) const;
```

Comment:
I'm not sure whether this method is useful.

Reply:
I see the following code block repeating a lot:
```
auto state = space->createState();
space->convertPositionsToState(position, state);
```
Figured I'd make a helper function.

Reply:
In that case, let's add it as a separate/future PR instead so this one is just about the IK rankers.